libgit2-sys/lib.rs

```diff
@@ -9,6 +9,8 @@ use libc::{c_char, c_int, c_uchar, c_uint, c_void, size_t};
 use libssh2_sys as libssh2;
 use std::ffi::CStr;
 
+pub const LIBGIT2_VERSION: &str = "1.0.0";
```

Comment:
Hm ideally this'd be auto-synthesized from libgit2 headers, would it be possible to defer this to later?

Reply:
This was only used for the test to match the version at the end. I changed it to do fuzzy matching.